client/my-sites/post-type-list/post-type-list-post-actions/index.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import EllipsisMenu from 'components/ellipsis-menu';
+import PostTypeListPostActionsEdit from './edit';
+import PostTypeListPostActionsPublish from './publish';
+import PostTypeListPostActionsTrash from './trash';
+import PostTypeListPostActionsView from './view';
+import PostTypeListPostActionsRestore from './restore';
+
+export default function PostTypeListPostActions( { globalId } ) {
+	return (
+		<div className="post-type-list-post-actions">
+			<EllipsisMenu position="bottom left">
+				<PostTypeListPostActionsView globalId={ globalId } />
```

Comment:
That seems sensible, as long as I can add children to the base set as well without setting it to false. The pages popover menu is a good example to keep in mind.